nitro/Dockerfile

```diff
@@ -13,53 +13,60 @@
 #       Each RUN contains a bundle of steps, which reduces the cache.
 
 FROM veracruz/base:latest
-ARG USER=root
-ARG UID=0
 ARG ARCH=x86_64
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    python-setuptools \
-    python3-setuptools \
-    python3 \
-    python3-pip \
-    musl-tools \
-    ca-certificates \
-    curl \
-    lxc \
-    iptables \
-    jq \
-    iproute2 \
-    apt-transport-https \
-    gnupg \
-    lsb-release \
-    software-properties-common
-
-RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
-RUN add-apt-repository "deb https://download.docker.com/linux/ubuntu bionic stable"
-RUN apt-get update && apt-get install --no-install-recommends -y docker-ce-cli
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+        apt-transport-https \
+        ca-certificates \
+        gnupg \
+        iproute2 \
+        iptables \
+        jq \
+        lsb-release \
+        lxc \
+        musl-tools \
+        python3 \
+        python3-pip \
+        python3-setuptools \
+        software-properties-common ; \
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - ; \
+    add-apt-repository "deb https://download.docker.com/linux/ubuntu bionic stable" ; \
+    apt-get update && apt-get install --no-install-recommends -y docker-ce-cli ; \
+    apt-get autoremove -y && apt-get clean ; \
+    rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
 
 COPY aws-nitro-enclaves-cli/build/nitro_cli/${ARCH}-unknown-linux-musl/release/nitro-cli /usr/bin/
 
-RUN pip install -U setuptools && \
-    pip install awscli
+RUN pip install -U setuptools pip && \
+    pip install awscli && \
+    pip cache purge
```

Comment:
Would it accomplish the same thing to use `--no-cache-dir` instead of purging afterward?

Reply:
Probably. I did not know about `--no-cache-dir`